packages/cli/src/commands/curl/README.md

```diff
@@ -23,14 +23,15 @@ vercel curl <path> [options] [-- <curl-args>]
 - **`path`** (required): The API path to request
   - Can start with `/` (e.g., `/api/hello`) or without (e.g., `api/hello`)
   - Automatically normalized to include leading slash
+  - Must be a relative path — do not pass a full URL. To target a specific deployment base URL, use `--deployment <id>`
```

Comment:
Why can't they just pass the full url? At which point, also providing a `--deployment` option should be considered invalid